packages/resource-deployment/scripts/add-tags-for-batch-vmss.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set -eo pipefail
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -v <vmss name> -r <vmss resource group> -b <batch resource group name>
+"
+    exit 1
+}
+
+addResourceGroupNameTagToVMSS(){
+    az resource update --set tags.ResourceGroupName="$resourceGroupName" -g "$vmssResourceGroup" -n "$vmssName" --resource-type "Microsoft.Compute/virtualMachineScaleSets"
+
+    echo "Tag ResourceGroupName=$resourceGroupName was added to $vmssName vmss under $vmssResourceGroup resource group"
+}
+
+# Read script arguments
+while getopts ":v:r:b:" option; do
+    case $option in
+    v) vmssName=${OPTARG} ;;
```

Comment:
can we generalize it more; 
- Send the resource type too as an input
- Rename the file to "add-tags-for-resource.sh"

Reply:
we can generalize it further when we need it for other case. I am also doing some batch specific thing here.

Reply:
I only mean for adding tags, if we add the resource type as an input, we can use the same script for adding the resourceGroup name as a tag for the vmms